package.json

```diff
@@ -79,8 +79,9 @@
     "spec_to_console": "scripts/use_node scripts/spec_to_console",
     "pkg-version": "scripts/use_node -e \"console.log(require('./package.json').version)\"",
     "cypress:run-without-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=false",
-    "cypress:run-with-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=true,openSearchUrl=https://localhost:9200,WAIT_FOR_LOADER_BUFFER_MS=500"
- 
+    "cypress:run-with-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=true,openSearchUrl=https://localhost:9200,WAIT_FOR_LOADER_BUFFER_MS=500",
+    "osd:ciGroup10": "echo \"dashboard_sanity_test_spec.js\"",
+    "osd:ciGroup11": "echo \"apps/vis_builder/*.js\""
```

Comment:
Created an issue to address this concern https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6083

Reply:
Yea have we specified how we group the cypress tests within the dashboard repo into cigroups?

Reply:
Currently in Dashboards repo we are grouping it similar to how we group it in FT repo using package json by defining folder structure in it. I'll have to look into better way of doing it or something similar to how selenium tests are doing it today by passing it as a command.